inc/switchtec/switchtec.h

```diff
@@ -719,7 +719,7 @@ enum switchtec_fw_dlstatus {
 	SWITCHTEC_DLSTAT_COMPLETES = 7,
 	SWITCHTEC_DLSTAT_SUCCESS_FIRM_ACT = 8,
 	SWITCHTEC_DLSTAT_SUCCESS_DATA_ACT = 9,
-	SWITCHTEC_DLSTAT_DOWNLOAD_TIMEOUT = 14,
+	SWITCHTEC_DLSTAT_DOWNLOAD_TIMEOUT = 0xe,
```

Comment:
If you're going to use hex, you should change all the above to use hex as well, to make it consistent.

Reply:
Updated.